test/Microsoft.Azure.Devices.Edge.Test.Common/DaemonConfiguration.cs

```diff
@@ -4,15 +4,16 @@ namespace Microsoft.Azure.Devices.Edge.Test.Common
 {
     using System;
     using System.IO;
+    using Microsoft.Azure.Devices.Edge.Test.Common.Certs;
 
     public class DaemonConfiguration
     {
         readonly string configYamlFile;
         readonly YamlDocument config;
 
-        public DaemonConfiguration()
+        public DaemonConfiguration(string configYamlFile)
         {
-            this.configYamlFile = Platform.GetConfigYamlPath();
+            this.configYamlFile = configYamlFile;
```

Comment:
```suggestion
            Precondition.CheckArgument(File.Exists(configYamlFile), $"Given config yaml file '{configYamlFile}' doesn't exist");
            this.configYamlFile = configYamlFile;
```

Reply:
The next line in the ctor (the line that tries to read the file) will throw basically that same exception, so I don't think it's necessary.

```
Unhandled Exception: System.IO.FileNotFoundException: Could not find file 'C:\Users\damonb\projects\temppp\whatever'.
```